types.go

```diff
@@ -10,18 +10,18 @@ type CbDevice struct {
 	Credentials        []CbDeviceCredential  `json:"credentials"`
 	GatewayConfig      GatewayConfig         `json:"gatewayConfig,omitempty"`
 	Id                 string                `json:"id"`
-	LastConfigAckTime  string                `json:"lastConfigAckTime,omitempty"`
-	LastConfigSendTime string                `json:"lastConfigSendTime,omitempty"`
-	LastErrorStatus    DeviceLastErrorStatus `json:"lastErrorStatus"`
-	LastErrorTime      string                `json:"lastErrorTime,omitempty"`
-	LastEventTime      string                `json:"lastEventTime,omitempty"`
-	LastHeartbeatTime  string                `json:"lastHeartbeatTime,omitempty"`
-	LastStateTime      string                `json:"lastStateTime,omitempty"`
-	LogLevel           string                `json:"logLevel,omitempty"`
-	Metadata           map[string]string     `json:"metadata,omitempty"`
+	LastConfigAckTime  string                `json:"-"`
+	LastConfigSendTime string                `json:"-"`
+	LastErrorStatus    DeviceLastErrorStatus `json:"-"`
```

Comment:
I changed the encoding to - mainly so that LastErrorStatus and State aren't included in the post body for createDevice. The new version of the webhook will return a 400 since those are output-only fields. @ms1752 @rajasd27 Are we cool with not migrating all of these timestamps? IMO it makes sense because those things happened in GCP